lib/neo4j/core/cypher_session/adaptors/http.rb

```diff
@@ -94,12 +95,12 @@ class Requestor
             default_url('http://neo4:neo4j@localhost:7474')
             validate_uri { |uri| uri.is_a?(URI::HTTP) }
 
-            def initialize(url, user_agent_string, instrument_proc)
+            def initialize(url, user_agent_string, instrument_proc, adaptor)
               self.url = url
               @user = user
               @password = password
               @user_agent_string = user_agent_string
-              @faraday = faraday_connection
+              @faraday = faraday_connection(adaptor)
```

Comment:
Rather than adding a specific `adaptor` argument, what about a `faraday_options` argument that could simply be passed through so that people could do configuration of other faraday options?